survey/HTML-syntax-in-EPUB.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Survey on the Impact of Allowing HTML Syntax in EPUB
+date: 2025-05-08
+---
+
+The Publishing Maintenance Working Group (PMWG) is exploring the possibility of allowing HTML as an alternative content document format in EPUB 3.4, alongside the existing XHTML format.
```

Comment:
```suggestion
The Publishing Maintenance Working Group (PMWG) is exploring the possibility of allowing the [HTML syntax](https://html.spec.whatwg.org/multipage/syntax.html#syntax) in the HTML content document format in EPUB 3.4, and not restricting it to the [XML Syntax](https://html.spec.whatwg.org/multipage/xhtml.html#the-xhtml-syntax) as it is in earlier versions of EPUB 3.
```

The reference to the "syntax" was mentioned on the call...

Reply:
It might help to explain what we're talking about before jumping into the new formats we're allowing, and we should use xml and html syntaxes consistently otherwise we're just going to sow confusion. I'd suggest a simple intro like:

HTML(5) defines both XML and HTML syntaxes for authoring documents. EPUB 3, however, currently restricts authors to using only the XML syntax for content documents. The Publishing Maintenance Working Group (PMWG) is considering expanding EPUB content to allow both syntaxes to be used to author content documents in EPUB 3.4.